terraform/aks/app.tf

```
@@ -1,38 +1,36 @@
-resource "kubernetes_job" "migrations" {
-  metadata {
-    name      = "${var.service_name}-${local.app_name_suffix}-migrations"
-    namespace = var.namespace
-  }
+module "migrations_job_configuration" {
+  source = "./vendor/modules/aks//aks/application_configuration"
+
+  namespace              = var.namespace
+  environment            = local.app_name_suffix
+  azure_resource_prefix  = var.azure_resource_prefix
+  service_short          = var.service_short_name
+  config_short           = var.environment_short_name
+  secret_key_vault_short = "api"
```

Might be better to use the `inf` secret here instead of `api`.
```suggestion
  secret_key_vault_short = "inf"
```

---

done.